lib/l10n/app_he.arb

```diff
@@ -198,12 +198,16 @@
   "@explorerAppBarTitle": {},
   "explorerFlagInfoDialogTitle": "סימון חבר",
   "@explorerFlagInfoDialogTitle": {},
-  "explorerFlagInfoDialogSubTitlePartTwo": "כשמישהh/ו מסומנ/ת, תדמית המשתמש שלו תרד: מספיק סימונים ותדמית המשתמש שלהם תרד לאפס. הסימון גם משפיע על כל החברים שקיבלו ערבות מהחבר המסומן אז שימ/י לב את מי את/ה מסמנ/ת!",
+  "explorerFlagInfoDialogSubTitlePartTwo": "כשמישהי/ו מסומנ/ת, תדמית המשתמש שלו תרד: מספיק סימונים ותדמית המשתמש שלהם תרד לאפס. הסימון גם משפיע על כל החברים שקיבלו ערבות מהחבר המסומן אז שימ/י לב את מי את/ה מסמנ/ת!",
```

Comment:
Where are this changes coming from lol

Reply:
mmm weblate ??

Reply:
@gguij004 These are from Weblate, from the person who's contributing Hebrew translations 🤓

Reply:
yeah but why are they showing on this pr and the other one you had

Reply:
I guess I started from the beginning of the release/2.8.0 and I brought the translations of webalte